addon/templates/components/power-select-multiple/trigger.hbs

```diff
@@ -16,8 +16,8 @@
       {{/if}}
     </li>
   {{else}}
-    {{#if (and placeholder (not searchEnabled))}}
-      <span class="ember-power-select-placeholder">{{placeholder}}</span>
+    {{#if (not-eq placeholder maybePlaceholder)}}
```

Comment:
This is changing the logic and concerns me. 
Originally the placeholder only shows if the search was not enabled but if I'm reading this right in here it's going to show even if there is a search input. Am I wrong?